exercises/allergies/canonical-data.json

```diff
@@ -15,7 +15,9 @@
         {
           "description": "no allergies means not allergic",
           "property": "allergicTo",
-          "score": 0,
+          "input": {
+            "score": 0
```

Comment:
Does "input" really need "score"?  Why not just:
```text
"input": 0,
```
Of course like changes for other cases.  Unless we are really married to "score".  Others should perhaps weigh in on this point.

Reply:
don't care what y'all decide, but https://github.com/exercism/problem-specifications/issues/996#issuecomment-342740372 and https://github.com/exercism/problem-specifications/issues/996#issuecomment-342751835 are explicitly in favour of having `"input": {"score": 0}` not `"input": 0` therefore that is the only thing I support in https://github.com/exercism/problem-specifications/pull/998.

Reply:
I missed that detail.